### PR TITLE
nbc#757132 and bnc#777449 puzzle string

### PR DESCRIPTION
**Pull request by varkoly**
Please review and check in.

### CODE REVIEW

File: src/LdapDatabase.ycp [hunk 1]
```diff
@@ -323,8 +323,7 @@
                 {
                     map<string, string> err = LdapServer::ReadError();
                     boolean res = Popup::AnyQuestion(Label::WarningMsg(),
-                                        _("The Base Object: \"") + db["suffix"]:"" +
-                                        _("\" can not be auto created by YaST.\n") +
+                                        sformat(_("The Base Object: \"%1\" can not be auto created by YaST.\n"), db["suffix"]:"") +
```
Comment (mchf) [inline src/LdapDatabase.ycp]: can not -> cannot

Comment (mchf) [inline src/LdapDatabase.ycp]: why is error msg added with '+' when converting to sformat?

**Comment by varkoly** [comment]
fix

**Comment by varkoly** [comment]
Add changes

**Comment by mvidner** [comment]
Merged manually in c15e57e96a7b114896372ad35c3ea5756bf72d47

**Pull request merged**